ROBOT:
  MUTEX_PREFIX : "kict_mp_camera002_"

